config/redirects

```diff
@@ -31,39 +34,39 @@ raw: ${prefix}/sparkR -> ${base}/v3.0/r-api/
 [*-v1.1]: ${prefix}/${version}/scala-api -> ${base}/${version}/getting-started/
 [*-v1.1]: ${prefix}/${version}/r-api -> ${base}/${version}/sparkR/
 (v1.1-v3.0]: ${prefix}/${version}/getting-started -> ${base}/${version}/
-(v1.1-*]: ${prefix}/${version}/spark-sql -> ${base}/${version}/
+(v1.1-master]: ${prefix}/${version}/spark-sql -> ${base}/${version}/
```

Comment:
I'm not sure, but I noticed that the rest of the driver redirects files use this syntax (rather than `-*]`) so I changed it to be safe. Happy to change it back though.

Reply:
I think it is ok either way. I was just wondering if there was any advantage.
Hope the question wasn't confusing since markdown interpreted the ``*`` as the italics marker.